saleor/checkout/fetch.py

```diff
@@ -56,6 +54,41 @@ class CheckoutLineInfo(LineInfo):
     discounts: list["CheckoutLineDiscount"]
     tax_class: Optional["TaxClass"] = None
 
+    @cached_property
+    def variant_discounted_price(self) -> Money:
+        """Return the discounted variant price.
```

Comment:
Maybe we should specify that this price include just catalogue promotion

Reply:
Added explanation in line below. Thx 👍